content/Teams/onboarding/README.md

```diff
@@ -0,0 +1,444 @@
+
+1. Read the above doc carefully
+2. Create your Bugzilla account
+3. Request and ensure the necessary permissions are granted (ask your manager)
+4. Read the [OpenShift Bugzilla Process documentation](https://source.redhat.com/groups/public/atomicopenshift/atomicopenshift_wiki/openshift_bugzilla_process)
+5. Make sure you are aware to hit “show advanced fields” for doc text fields and others. Alternatively you can set it in *Preferences > General Preferences > Initially hide the advanced fields in bug editor: Off*. (The fields "Doc Type" and "Doc Text" are hidden otherwise. If these fields are not set, one will get emails about "Action required for your OpenShift Container Platform bugs - Missing doc text" on otherwise resolved issues.)
+
+### Spinning up OpenShift clusters
+
+There are several ways to create an OpenShift cluster or to use an existing one. You can choose the method which is most suitable for your particular use-case.
+
+* There is a global OpenShift cluster which gets recycled daily. The credentials for accessing it are posted in the [forum-ui-clusters](https://coreos.slack.com/archives/CUNC2JA2U) channel.
+* Short-lived clusters (up to 2 hours) can be provisioned using the **cluster-bot** Slack app. Type in *help* for instructions on how to interact with cluster-bot.
+* Clusters which last up to a day can be created using the CLI. Review the documentation for [AWS](https://docs.google.com/document/d/1j7bhLXT_cIAjpMh_x2jeegtpE7495Mj5A-EcQsgZEDo/edit#) and [GCP](https://docs.google.com/document/d/1qm37EKkjgoPtjW4909UClzvsjQO5VSpPUvFO_hW_PEg) and request your accounts. Note: for GCP the project name is openshift-gce-devel
+
+### Administrative things
+
+1. Review all the onboarding instructions/documentation sent to your email by the People Team (HR).
+2. Submit an[Employee Badge Request Form](https://redhat.service-now.com/rh_ess/cat_item.do?&sysparm_document_key=sc_cat_item,ac8bb875a82282004c7185ae62325874)
+3. Register for new hire orientation NHO (you will get an email about this from HR)
+4. Speak with your manager about any additional equipment you need (e.g. keyboard, mouse, etc)
+5. Login to Rover (our company directory) and update your profile [https://rover.redhat.com/people/profile/](https://rover.redhat.com/people/profile/). Be sure to do the following:
+   * Add a short bio
+   * Bookmark Rover for future reference
+6. Add yourself in [https://spaces.redhat.com/display/OBS/Monitoring+Team](https://spaces.redhat.com/display/OBS/Monitoring+Team)
+
+### Meetings
+
+Ask your manager to be added to the [Observability Program](https://calendar.google.com/calendar/u/0?cid=cmVkaGF0LmNvbV91N3YwbGt2cnRuM2wwbWJmMnF2M2VkMm12MEBncm91cC5jYWxlbmRhci5nb29nbGUuY29t) calendar. Ensure you attend the following recurring meetings:
+
+* Team syncs
+* Sprint retro/planning
+* Sprint reviews
+* Weekly architecture call
```

Comment:
we don't do this anymore but we have office hours for Observatorium, prometheus operator and observability operator (at least)